frontend/packages/console-app/src/components/console-operator/ConsolePluginManifestPage.tsx

```diff
@@ -14,10 +16,25 @@ export const ConsolePluginManifestPage: React.FC<PageComponentProps> = ({ obj })
     pluginName,
   ]);
 
+  const manifestJson = React.useMemo(() => {
+    return pluginManifest ? JSON.stringify(pluginManifest, null, 2) : '';
+  }, [pluginManifest]);
+
   return (
     <PaneBody>
       {pluginManifest ? (
-        <CopyToClipboard value={JSON.stringify(pluginManifest, null, 2)} />
+        <CodeEditor
+          value={manifestJson}
```

Comment:
IMO, we don't need the YAML language support that `CodeEditor` provides so we can use the more basic one

```suggestion
        <BasicCodeEditor
          code={manifestJson}
          isFullHeight
```